### ISSUE TITLE
Notes management is confusing/buggy

### ISSUE THREAD

**Issue by SolarLiner** [Philip-Scott/Notes-up#266]
I've downloaded the app from the AppCenter to try it out as I also do take a lot of notes for school (but not only), however I find managing those notes rather confusing. Or rather I think it is buggy.

There is no way to rename a note from what I tried (double click, right-click, long press), and there is no feedback when deleting a note; rather it is duplicated into a "trash" section.

Deleting notebooks is as confusing, as deleting also copies the notebook to the trash section while leaving it in the list of active notebooks.

This happened on first run after downloading from the AppCenter.

I really wish to use this nifty little app and can see the usefulness - however it is not usable for me quite yet.

**Comment by Wetrain** [comment]
@SolarLiner Regarding the deletion of notes, i am also experiencing this issue

**Comment by andbuitra** [comment]
Yes, this issue still exists on latest build for Juno. Moving notes (in a nested form) is really buggy. The deletion of Notebook moves it to trash **while keeping it on the main view** and get only deleted _from both locations_ after restarting the app.
